src/plinder/data/pipeline/transform.py

```diff
@@ -133,14 +133,15 @@ def calc_pchembl(affinity: float) -> Any:
         "Ligand HET ID in PDB",
         "PDB ID(s) for Ligand-Target Complex",
         "Ki (nM)",
-        "IC50 (nM)",
+        # "IC50 (nM)",
         "Kd (nM)",
         "EC50 (nM)",
     ]
     df = pd.read_csv(raw_affinity_path, sep="\t", usecols=cols, low_memory=False)
 
     df["pchembl"] = (
-        df[["Ki (nM)", "IC50 (nM)", "Kd (nM)"]]
+        # df[["Ki (nM)", "IC50 (nM)", "Kd (nM)"]]
```

Comment:
@yusuf1759 is this the right way to drop IC50 from bindingdb? Anything else needed for papyrus?

Reply:
Seems like papyrus would include EC50 and IC50 data 😱

<img width="794" alt="image" src="https://github.com/user-attachments/assets/19b36815-fde0-421b-b2de-344796e60dd0">

Although with some curation modification - but it is still 😱 according to this
https://pubs.acs.org/doi/10.1021/acs.jcim.4c00049

Will need to review this to see if this makes sense, maybe we should just map to curated datasets from polaris?
https://polarishub.io/datasets?accessTypes=public

Reply:
We could also keep this `bindingDB` so far, comment out `papyrus` to avoid inconsistency issues, and see what we do with this once we have a full strategy on how to potentially integrate `polaris` or `bigbind` sets.

Reply:
Commented out `papyrus` in [f022c83](https://github.com/plinder-org/plinder/pull/14/commits/f022c838b47003899d6c1a6e3465c3a84ad1a964) - leaves only `bindingDB` with "Ki (nM)" and "Kd (nM)" data

Reply:
[a385a41](https://github.com/plinder-org/plinder/pull/14/commits/a385a41daf3c37a5adb09eb0b6a81d215cc0ed8d) - changed to (3, 50)